diff --git a/node_modules/react-native-worklets/android/CMakeLists.txt b/node_modules/react-native-worklets/android/CMakeLists.txt
index 7ce6058..2f2d88f 100644
--- a/node_modules/react-native-worklets/android/CMakeLists.txt
+++ b/node_modules/react-native-worklets/android/CMakeLists.txt
@@ -1,5 +1,7 @@
-project(Worklets)
 cmake_minimum_required(VERSION 3.8)
+# Must be set before `project()` so CMake uses this value for object paths.
+set(CMAKE_OBJECT_PATH_MAX 1024)
+project(Worklets)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
 
